packages/shared/src/components/modals/bookmark/BookmarkFolderModal.tsx

```diff
@@ -163,14 +163,23 @@ const BookmarkFolderModal = ({
                 key={emoji}
                 onClick={() => setIcon(emoji)}
                 className={classNames(
-                  'size-12',
+                  'aspect-square !size-12',
```

Comment:
Shouldn't size already force equal? (this is fine though, just wondering)

Reply:
Yes, `aspect-square` was a test, but not needed indeed.